packages/framework/src/Component/ClassExtension/TypehintHelper.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\ClassExtension;
+
+use Roave\BetterReflection\Reflection\ReflectionNamedType;
+use Roave\BetterReflection\Reflection\ReflectionProperty;
+
+class TypehintHelper
+{
+    /**
+     * @param \Roave\BetterReflection\Reflection\ReflectionProperty $reflectionProperty
+     * @return string
+     */
+    public static function getPropertyTypeFromTypehint(ReflectionProperty $reflectionProperty): string
+    {
+        $type = $reflectionProperty->getType();
+
+        if (($type instanceof ReflectionNamedType) === false) {
+            return '';
```

Comment:
I prefer to return `null` in this method and then use `$propertyTypeString ?? ''` in PropertyAnnotationFactory. 

Right now the  `replaceInPropertyType` method seems kinda weird with mixing comparison of null and empty string